platform/viewer/src/routes/Mode/Mode.tsx

```diff
@@ -15,10 +15,11 @@ import Compose from './Compose';
  * @param props.servicesManager to read services from
  * @param props.studyInstanceUIDs for a list of studies to read
  * @param props.dataSource to read the data from
+ * @param props.seriesInstanceUID query param to read the data from
  * @returns array of subscriptions to cancel
  */
 function defaultRouteInit(
-  { servicesManager, studyInstanceUIDs, dataSource },
+  { servicesManager, studyInstanceUIDs, dataSource, seriesInstanceUID },
```

Comment:
Grab the filters here direct, as there might be more than just seriesInstanceUID